frontend/src/scenes/max/maxTypes.ts

```diff
@@ -11,6 +13,96 @@ export enum MaxContextType {
 
 export type InsightWithQuery = Pick<Partial<QueryBasedInsightModel>, 'query'> & Partial<QueryBasedInsightModel>
 
+// Simplified product information for Max context
+export interface MaxProductInfo {
+    type: string
+    name: string
+    description: string
+    is_used: boolean // current_usage > 0
+    has_exceeded_limit: boolean
+    current_usage?: number
+    usage_limit?: number | null
+    percentage_usage: number
+    custom_limit_usd?: number | null
+    next_period_custom_limit_usd?: number | null
+    docs_url?: string
+}
+
+// Simplified addon information for Max context
+export interface MaxAddonInfo {
+    type: string
+    name: string
+    description: string
+    is_used: boolean // current_usage > 0
+    has_exceeded_limit: boolean
+    current_usage: number
+    usage_limit?: number | null
+    percentage_usage?: number
+    custom_limit_usd?: number | null
+    next_period_custom_limit_usd?: number | null
+    docs_url?: string
+}
+
+// Usage data context for Max
+export interface MaxUsageContext {
+    date_range: {
+        start_date: string
+        end_date: string
+    }
+    usage_summary: Array<{
+        product_type: string
+        product_name: string
+        total_usage: number
+        dates: string[]
+        data: number[]
+    }>
+}
+
+export interface MaxBillingContext {
+    // Overall billing status
+    has_active_subscription: boolean
+    subscription_level: 'free' | 'paid' | 'custom'
+    billing_plan: string | null
+    is_deactivated?: boolean
+
+    // Products information
+    products: MaxProductInfo[]
+
+    // Addons information (flattened from all products)
+    addons: MaxAddonInfo[]
+
+    // Usage summary
+    total_current_amount_usd?: string
+    total_projected_amount_usd?: string
```

Comment:
We also have current/projected amounts with limit - might be helpful, even though we have it per product